fbw-a32nx/src/wasm/systems/a320_systems_wasm/src/payload.rs

```diff
@@ -39,6 +39,22 @@ pub(super) fn payload(builder: &mut MsfsAspectBuilder) -> Result<(), Box<dyn Err
         Variable::aircraft("PAYLOAD STATION WEIGHT", "Pounds", 8),
         Variable::aspect("PAYLOAD_STATION_8_REQ"),
     );
+    builder.copy(
+        Variable::aircraft("INTERACTIVE POINT OPEN", "percent", 0),
+        Variable::aspect("INTERACTIVE_POINT_0_OPEN_REQ"),
+    );
+    builder.copy(
+        Variable::aircraft("INTERACTIVE POINT OPEN", "percent", 1),
+        Variable::aspect("INTERACTIVE_POINT_1_OPEN_REQ"),
+    );
+    builder.copy(
+        Variable::aircraft("INTERACTIVE POINT OPEN", "percent", 2),
+        Variable::aspect("INTERACTIVE_POINT_2_OPEN_REQ"),
+    );
+    builder.copy(
+        Variable::aircraft("INTERACTIVE POINT OPEN", "percent", 3),
+        Variable::aspect("INTERACTIVE_POINT_3_OPEN_REQ"),
+    );
```

Comment:
You actually don't need this. You can add calls to `provides_aircraft_variable` (see lib.rs:290)